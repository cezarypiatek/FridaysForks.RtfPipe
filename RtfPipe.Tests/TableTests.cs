using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace RtfPipe.Tests;

[TestClass]
public class TableTests
{
  [TestMethod]
  public void marge_scenario_1() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_2() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_3() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_4() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_5() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_6() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_7() => this.VerifyCurrentScenario();
}
