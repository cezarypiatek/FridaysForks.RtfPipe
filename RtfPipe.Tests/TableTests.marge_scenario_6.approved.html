<div style="font-size:12pt;font-family:&quot;Times New Roman&quot;, serif;">
    <table style="font-family:Cambria, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0 0 0 0.7px;">
        <colgroup>
            <col style="width:101.8px;" />
            <col style="width:72.9px;" />
            <col style="width:81.3px;" />
            <col style="width:84px;" />
            <col style="width:84px;" />
            <col style="width:148px;" />
        </colgroup>
        <tr style="text-align:left;height:21px;">
            <td rowspan="3" style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;background:#F2F2F2;border-top:1px solid;border-right:none;border-bottom:1px solid #000000;border-left:1px solid;padding:0 4.7px;">
                <strong>Rodzaj paliwa
                </strong>
            </td>
            <td colspan="2" style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:1px solid;border-right:1px solid #000000;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">
                <strong>Ogrzewanie
                </strong>
            </td>
            <td colspan="2" style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:1px solid;border-right:1px solid #000000;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <strong>CWU
                </strong>
            </td>
            <td rowspan="2" style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;background:#F2F2F2;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <strong>EP
                </strong>
            </td>
        </tr>
        <tr style="text-align:left;height:24px;">
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">Q
                <sub>K,H
                </sub>
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Q
                <sub>P,H
                </sub>
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Q
                <sub>K,W
                </sub>
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Q
                <sub>P,W
                </sub>
            </td>
        </tr>
        <tr style="text-align:left;height:24px;">
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">[kWh/rok]
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">[kWh/rok]
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">[kWh/rok]
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">[kWh/rok]
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">[kWh/(m
                <sup>2
                </sup>*rok)]
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;height:20px;">
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;vertical-align:bottom;background:#F2F2F2;border-top:none;border-right:none;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">
                <strong>Pellet
                </strong>
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">9496,34
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">3108,99
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">1821,35
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">1373,53
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <span style="font-family:Arial, sans-serif;">23,34
                </span>
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;height:21px;">
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;vertical-align:bottom;background:#F2F2F2;border-top:1px solid;border-right:none;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">
                <strong style="font-family:&quot;Arial CE&quot;, sans-serif;">Pompa ciepła
                </strong>
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">2884,07
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">3228,58
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">657,52
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">1469,52
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;vertical-align:bottom;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <span style="font-family:Arial, sans-serif;">24,47
                </span>
            </td>
        </tr>
    </table>
 </div>